src/docs/development/ui/advanced/gestures.md

```diff
@@ -6,134 +6,183 @@ This document explains how to listen for, and respond to,
 _gestures_ in Flutter. Examples of gestures include
 taps, drags, and scaling.
 
-The gesture system in Flutter has two separate layers.  The first layer has raw
-pointer events, which describe the location and movement of pointers (e.g.,
-touches, mice, and styli) across the screen. The second layer has _gestures_,
-which describe semantic actions that consist of one or more pointer movements.
+The gesture system in Flutter has two separate layers.
+The first layer has raw pointer events that describe
+the location and movement of pointers (for example,
+touches, mice, and styli) across the screen.
+The second layer has _gestures_ that describe semantic
+actions that consist of one or more pointer movements.
 
 ## Pointers
 
-Pointers represent raw data about the user's interaction with the device's
-screen.  There are four types of pointer events:
-
-- [`PointerDownEvent`]({{site.api}}/flutter/gestures/PointerDownEvent-class.html)
-  The pointer has contacted the screen at a particular location.
-- [`PointerMoveEvent`]({{site.api}}/flutter/gestures/PointerMoveEvent-class.html)
-  The pointer has moved from one location on the screen to another.
-- [`PointerUpEvent`]({{site.api}}/flutter/gestures/PointerUpEvent-class.html)
-  The pointer has stopped contacting the screen.
-- [`PointerCancelEvent`]({{site.api}}/flutter/gestures/PointerCancelEvent-class.html)
-  Input from this pointer is no longer directed towards this app.
-
-On pointer down, the framework does a _hit test_ on your app to determine which
-widget exists at the location where the pointer contacted the screen. The
-pointer down event (and subsequent events for that pointer) are then dispatched
-to the innermost widget found by the hit test. From there, the events bubble up
-the tree and are dispatched to all the widgets on the path from the innermost
-widget to the root of the tree. There is no mechanism for canceling or stopping
-pointer events from being dispatched further.
+Pointers represent raw data about the user's interaction
+with the device's screen.
+There are four types of pointer events:
+
+[`PointerDownEvent`][]
```

Comment:
The formatting is off in https://sz-flutter.firebaseapp.com/docs/development/ui/advanced/gestures#pointers. I rather like the old bullets if you add `:`s to separate the definitions from the descriptions.